content/docs/operating/integrations.md

```diff
@@ -50,7 +50,7 @@ For notification mechanisms not natively supported by the Alertmanager, the
   * [Phabricator / Maniphest](https://github.com/knyar/phalerts)
   * [SMS](https://github.com/messagebird/sachet): supports [multiple providers](https://github.com/messagebird/sachet/blob/master/examples/config.yaml)
   * [Telegram bot](https://github.com/inCaller/prometheus_bot)
-  * [prom2teams](https://github.com/idealista/prom2teams): forwards received alerts to [Microsoft Teams](http://teams.microsoft.com) using defined connectors
+  * [prom2teams](https://github.com/idealista/prom2teams): forwards alerts to Microsoft Teams
```

Comment:
notifications, not alerts.

Keep it sorted.

Reply:
done